Web Development/Random Anime profile/README.md

```diff
@@ -0,0 +1,31 @@
+# Project Name: 
+
+- Random Anime profiles
+  
+# Preview: 
+![anime png](https://user-images.githubusercontent.com/95171638/166420861-780c6682-1049-4c82-9823-021a6d6309d6.png)
+
```

Comment:
Add a working video too in this section.

Reply:
@manisha-27 changes done please check it again.